tensorflow/python/ops/nn_ops.py

```diff
@@ -805,9 +805,14 @@ def op(converted_input, _, converted_padding):  # pylint: disable=missing-docstr
         filter_shape=window_shape)
 
 
-def atrous_conv2d(value, filters, rate, padding, name=None):
+def atrous_conv2d(value, filters, rate, padding, strides=None, name=None):
```

Comment:
Not clear if it makes sense to add strides, since it is supported by the more generic convolution anyway, but if it is added, it should be documented in the Args section.